temporalio/client.py

```diff
@@ -3233,7 +3233,7 @@ class ScheduleActionStartWorkflow(ScheduleAction):
     headers: Optional[Mapping[str, temporalio.api.common.v1.Payload]]
 
     @staticmethod
-    def _from_proto(
+    def _from_proto(  # pyright: ignore
```

Comment:
Hrmm, I wonder if this is the same thing we ignore in the system-agnostic way below and pyright just cares about the line the declaration started instead of the param.